simulator/docs/record-and-replay-cluster-events.md

```diff
@@ -0,0 +1,95 @@
+# [Beta] Record your real cluster's events and replay them in the simulator
+
+You can record events from your real cluster and replay them in the simulator. This feature is useful for reproducing issues that occur in your real cluster.
```

Comment:
Please rephrase "events" in this doc. It's unclear what it means for first-readers.

Reply:
Rephrased it to "changes" or "changes in resources". How about it?

Reply:
Probably go more straightforward like "You can record resource addition/update/deletion at your real cluster"